src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -124,18 +141,16 @@ public boolean login(final String userName, final HashedPassword hashedPassword)
         }
       }
       // update last login time
-      try (
-          final PreparedStatement ps = con.prepareStatement("update ta_users set lastLogin = ? where username = ?")) {
-        ps.setTimestamp(1, new Timestamp(System.currentTimeMillis()));
+      try (final PreparedStatement ps = con.prepareStatement("update ta_users set lastLogin=? where username=?")) {
```

Comment:
formatting nit, space between the '?' and '=' was chomped

Reply:
This way the line was <=120 chars long